specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json

```diff
@@ -506,6 +617,97 @@
       },
       "x-nullable": false
     },
+    "Authorization": {
+      "description": "Request parameter that contains claims for authorizing an operation.",
+      "required": [
+        "modelId"
+      ],
+      "type": "object",
+      "properties": {
+        "modelId": {
+          "description": "Form Recognizer model identifier that will be used as a claim token to authorize the request.",
+          "type": "string",
+          "format": "uuid"
+        }
+      }
+    },
+    "CopyRequest": {
+      "description": "Request parameter to copy an existing custom model.",
+      "required": [
+        "targetResourceId",
+        "targetResourceRegion"
+      ],
+      "type": "object",
+      "properties": {
+        "targetResourceId": {
+          "description": "Azure Resource Id of the target Form Recognizer resource where the model is copied to.",
+          "maxLength": 2048,
+          "type": "string",
+          "pattern": "^/subscriptions/[^/]*/resourceGroups/[^/]*/providers/Microsoft.CognitiveServices/accounts/[^/]*$",
+          "x-ms-azure-resource": true
+        },
+        "targetResourceRegion": {
+          "description": "Location of the target Azure resource.",
+          "$ref": "../../../Common/ExtendedRegions.json#/parameters/AzureRegion"
```

Comment:
A property cannot reference a parameter definition. Suggest defining the AzureRegion enum and referencing the definition.

Reply:
The target resource region no more references any external type - to avoid maintenance issues. Defines a simple string pattern and length constraint.